ajax/libs/simple-hint/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name":"simple-hint",

```

Comment:
@rajpatel507 There should be a space after colon mark. Please fix all the fields. Thanks.